src/frontend/src/pages/LoginPage.vue

```diff
@@ -42,6 +42,14 @@ import Button from "../components/Button.vue";
     TextInput,
     Button,
   },
+  methods: {
```

Comment:
I think this got switched back when merged with staging, please move into export default class!